app/src/main/java/kr/sesac/aoao/android/dino/ui/NewDinoActivity.kt

```diff
@@ -69,4 +75,29 @@ class NewDinoActivity : AppCompatActivity(){
              else findViewById<View>(resourceId)?.visibility = View.INVISIBLE //선택하지 않은 색들 체크 표시 비활성화
         }
     }
+
+    /**
+     * 다이노 생성 통신
+     * @since 2024.01.26
+     * @author 김은서
+     */
+    private fun saveDino(accessToken : String){
+        val newDinoRequest = NewDinoRequest(dinoColor, dinoName.getText().toString())
+        NewDinoUtil.newDino(accessToken,
+                            newDinoRequest,
+                            this@NewDinoActivity,
+            onResponse = {
+                response ->
+                if(response.success){
+                    ToastGenerator.showShortToast("다이노 생성 완료!", this);
+                    val intent = Intent(this, RaiseDinoActivity::class.java)
+                    startActivity(intent);
+                    finish()
+                }
+            },
+            onFailure = {
+                ToastGenerator.showShortToast("서버 오류 발생. 다시 시도해주세요",this);
+            }
```

Comment:
```suggestion
            onFailure = { response -> 
                ToastGenerator.showShortToast("서버 오류 발생. 다시 시도해주세요",this);
                // response.message
            }
```
onFailure 에 response 매개변수를 받는다면 서버에서 보내주는 에러 메시지를 활용하여 메시지를 띄울 수 있습니다!! ㅎㅎ 참고만 해주세용

Reply:
헉! 좋은 코멘트 감삼당 서버 메시지 띄우는걸 ㄷ까먹었서🥹